src/views/Dashboard.vue

```diff
@@ -41,6 +41,7 @@
                   v-if="visibleForStaff"
                   addScrollbarOffset="true"
                   @close="reloadDashboardIfNeeded($event)"
+                  class="mt-n3"
```

Comment:
This is a different change that I didn't look at before 😛 

I see the H2 has `mb-3` (which would also work for your component here actually) but I still think it's better to have the lower element specify whitespace above it.

Is there any reason the bottom margin couldn't be removed from the header/h2, and top margin added to the element below the header?